chia/rpc/wallet_rpc_api.py

```diff
@@ -2473,19 +2472,19 @@ async def nft_count_nfts(self, request) -> EndpointResult:
             count = await self.service.wallet_state_manager.nft_store.count()
         return {"wallet_id": wallet_id, "success": True, "count": count}
 
-    async def nft_get_nfts(self, request) -> EndpointResult:
+    async def nft_get_nfts(self, request: Dict[str, Any]) -> EndpointResult:
         wallet_id = request.get("wallet_id", None)
         nfts: List[NFTCoinInfo] = []
         if wallet_id is not None:
             nft_wallet = self.service.wallet_state_manager.get_wallet(id=wallet_id, required_type=NFTWallet)
         else:
             nft_wallet = None
         try:
-            start_index = int(request.get("start_index"))
+            start_index = int(request.get("start_index", 0))
         except (TypeError, ValueError):
             start_index = 0
         try:
-            count = int(request.get("num"))
+            count = int(request.get("num", 50))
         except (TypeError, ValueError):
             count = 50
```

Comment:
the try-catch around these seem redundant now. If they are, I think they should be removed. if they aren't, it might warrant a comment, since it really *looks* suspicious.

I think this change is for the better, because it allows the removal of the try-catch blocks. But it also doesn't seem to be a typing change

Reply:
The statement will still fail if the value can't be converted to `int` so both, `TypeError` and `ValueError` are still possible? What is sad though is that we now have two places which define the default value but this is the least invasive change to not require coverage and there anyway should be a request schema which then can define the default in one place, which should be a different PR and be done basically for all RPC endpoints.

Reply:
if the value exists but cannot be interpreted as an `int`, that would seem to warrant a fatal error.

Reply:
but I suppose it was like this before you got here

Reply:
> but I suppose it was like this before you got here

Yes, basically nothing changed here in this regard.